casepro/backend/junebug.py

```diff
@@ -216,15 +214,10 @@ def pull_contacts(
 
         identities_to_update = list(chain(modified_identities, new_identities))
 
-        # all identities deleted in the Identity Store in the time window
-        deleted_identities = list(identity_store.get_identities(
-            optout_type='forget', updated_from=modified_after,
-            updated_to=modified_before))
-
         # the method expects fetches not lists so I faked it
         return sync_local_to_changes(
-            org, IdentityStoreContactSyncer(), [identities_to_update],
-            [deleted_identities], progress_callback)
+            org, IdentityStoreContactSyncer(), [identities_to_update], [],
```

Comment:
We should perhaps leave a comment here explaining why the set of deleted identities is empty (i.e. we delete things via the identity store callback).